src/DropdownMenu.js

```diff
@@ -16,11 +70,19 @@ const DropdownMenu = React.createClass({
         'dropdown-menu-right': this.props.pullRight
       };
 
+    // Remove outline on focus by default
+    let style = this.props.style || {};
+    style.outline = style.outline || 'none';
+
+    // Adding tabIndex -1 to allow for focus, but not tabable
     return (
         <ul
+          tabIndex="-1"
```

Comment:
Actually you know what each of those anchors should have `tabIndex='-1'` as per the W3C WAI-ARIA Spec. See: http://www.w3.org/WAI/PF/aria-practices/#menubutton